js/src/hooks/useGoogleMCAccount.js

```diff
@@ -46,15 +49,25 @@ const useGoogleMCAccount = () => {
 				};
 			}
 
-			const { getGoogleMCAccount, isResolving, hasFinishedResolution } =
-				select( STORE_KEY );
+			const selector = select( STORE_KEY );
+			const acc = selector[ googleMCAccountSelector ]();
+			const isResolvingGoogleMCAccount = selector.isResolving(
+				googleMCAccountSelector
+			);
+
+			const hasGoogleMCConnection = [
```

Comment:
```suggestion
			const hasGoogleMCConnection = ( googleMCAccount?.status === GOOGLE_MC_ACCOUNT_STATUS.CONNECTED ||
			( googleMCAccount?.status === GOOGLE_MC_ACCOUNT_STATUS.INCOMPLETE &&
				googleMCAccount?.step === 'link_ads' ) );
```

Reply:
Ref: https://github.com/woocommerce/google-listings-and-ads/blob/0383264758c76b6a4211ade8cfb81062751fb98f/js/src/setup-mc/setup-stepper/setup-accounts/index.js#L122-L129

Reply:
Synced in Slack with @joemcgill and @ankitrox . We'll keep existing implementation.